NOTICE

```diff
@@ -11,6 +11,13 @@ by Google in the Guava Library:
   * License: Apache License v2.0
   * Homepage: https://github.com/google/guava
 
+This product contains a modified portion of 'JsonUtf8Writer', distributed
+by Square, Inc in the Moshi Library:
+
+  * Copyright (C) 2010 Google Inc.
```

Comment:
this is actually correct even if it seems like copy/paste error https://github.com/square/moshi/blob/2cf3f0204c38d62f14d258bc671c010ae6f0ea83/moshi/src/main/java/com/squareup/moshi/JsonUtf8Writer.java